apps/aechannel/src/aesc_utils.erl

```diff
@@ -479,8 +479,8 @@ process_force_progress(Tx, Addresses,
     PoITrees0 = trees_from_poi(Addresses, PoI),
     Reserve = aesc_channels:channel_reserve(Channel),
     PoITrees =
-        try aesc_offchain_update:apply_on_trees(Update, PoITrees0, NextRound,
-                                                      Reserve)
+        try aesc_offchain_update:apply_on_trees(Update, PoITrees0, Trees, Env,
```

Comment:
Gas limit must be charged upfront, and the unused gas reimboursed. Please amend, or track ticket analogous to https://www.pivotaltracker.com/story/show/160671577 , or at least add comment in same mentioned ticket.

Reply:
Same as https://github.com/aeternity/epoch/pull/1592/files#r222612328 and https://github.com/aeternity/epoch/pull/1592/files#r222614101

So if confirmed that that "the off-chain update can use only the balance of the initiator and responder that is the off-chain trees i.e. balance that has already been taken out of the on-chain trees" then no changes needed.